### DISCUSSION TITLE
UNet Parameters: Kernel_size and up_kernel

### DISCUSSION THREAD

**Discussion by jmenville11** [Project-MONAI/MONAI#2798] [Q&A]
Hi, I saw in the docs that these two parameters must be odd integers, but are there any other constraints that apply to them? Either in relation to other model parameters or each other?

I’ve had training success with both set to 1 or 3 but not 5 or 7. Am hoping to better understand why. Thanks!

**Comment by Nic-Ma** [comment]
Hi @jmenville11 ,

You can check this tutorials for the details about how to set parameters for UNet:
https://github.com/Project-MONAI/tutorials/blob/master/modules/UNet_input_size_constrains.ipynb

Thanks.
